tools/pipelines/templates/build-npm-package.yml

```diff
@@ -1,598 +1,540 @@
-            inputs:
-              command: 'custom'
-              workingDir: ${{ parameters.buildDirectory }}
-              customCommand: 'run bundle-analysis:run'
-
-          - ${{ if and(or(eq(variables['Build.Reason'], 'IndividualCI'), eq(variables['Build.Reason'], 'BatchedCI')), eq(variables['System.TeamProject'], 'internal')) }}:
-            - task: Bash@3
-              displayName: 'List report.json'
-              inputs:
-                targetType: 'inline'
-                workingDirectory:  ${{ parameters.buildDirectory }}
-                script: |
-                  echo "Build Directory is ${{ parameters.buildDirectory }}";
-                  BUNDLE_SIZE_TESTS_DIR="${{ parameters.buildDirectory }}/artifacts/bundleAnalysis/@fluid-example/bundle-size-tests";
-                  echo "Contents of $BUNDLE_SIZE_TESTS_DIR:";
-                  ls -la $BUNDLE_SIZE_TESTS_DIR;
+                  targetType: inline
+                  workingDirectory: $(absolutePathToTelemetryGenerator)
+                  script: >
+                    echo "Creating output folder "
 
-            - template: include-telemetry-setup.yml
-              parameters:
-                devFeedUrl: $(ado-feeds-dev)
-                officeFeedUrl: $(ado-feeds-office)
+                    mkdir -p $(pipelineTelemetryWorkdir)/timingOutput
 
-            - task: Bash@3
-              displayName: Write bundle sizes measurements to Aria/Kusto
-              inputs:
-                targetType: 'inline'
-                workingDirectory: $(absolutePathToTelemetryGenerator)
-                script: |
-                  echo "Writing the following performance tests results to Aria/Kusto"
-                  echo "Report Size:"
-                  ls -la '../../examples/utils/bundle-size-tests/bundleAnalysis/report.json';
-                  node --require @ff-internal/aria-logger bin/run --handlerModule $(absolutePathToTelemetryGenerator)/dist/handlers/bundleSizeHandler.js --dir '../../artifacts/bundleAnalysis/@fluid-example/bundle-size-tests';
+                    echo "Retrieving pipeline run timeline data "
 
-        # Docs
-        - ${{ if ne(parameters.taskBuildDocs, false) }}:
-          - task: Npm@1
-            displayName: npm run ci:build:docs
-            inputs:
-              command: 'custom'
-              workingDir: ${{ parameters.buildDirectory }}
-              customCommand: 'run ci:build:docs'
+                    echo 'curl -u ":<REDACTED>"
+                    "https://dev.azure.com/fluidframework/internal/_apis/build/builds/$BUILD_ID/timeline"'
 
-        - ${{ if or(eq(parameters.publishDocs, true), eq(parameters.taskBuildDocs, true)) }}:
-          - task: PublishPipelineArtifact@1
-            displayName: Publish Artifact - _api-extractor-temp
-            inputs:
-              targetPath: '${{ parameters.buildDirectory }}/_api-extractor-temp'
-              artifactName: '_api-extractor-temp'
-              publishLocation: 'pipeline'
```

Comment:
Another instance of transforming the condition into a line that seems to not matter. But also, in this case the condition below this was also wrapped in single quotes, which I'm not sure will have the same runtime effect; I suspect not, so probably want to revert that bit.